src/devices/nm-device.c

```diff
@@ -2044,9 +2044,14 @@ nm_device_get_route_metric_default (NMDeviceType device_type)
 	 */
 
 	switch (device_type) {
-	/* 50 is also used for VPN plugins (NM_VPN_ROUTE_METRIC_DEFAULT) */
+
+	/* 50 is also used for VPN plugins (NM_VPN_ROUTE_METRIC_DEFAULT).
+	 *
+	 * Note that returning 50 from this function, means that this device-type is
```

Comment:
The comma after 'function' sounds bad. Also 'tread' -> 'treat' in commit subject.